fract4dgui/preferences.py

```diff
@@ -401,6 +401,23 @@ def set_prefs(*args):
 
         return widget
 
+    def create_continuous_zoom_widget(self):
+        widget = Gtk.CheckButton(label="Continuous _Zoom")
+        widget.set_tooltip_text("Activate continuous zoom to zoom in or out while keep clicked the mouse buttons")
+        widget.set_use_underline(True)
```

Comment:
tooltip_text and use_underline can be used as keyword arguments in Gtk.CheckButton() as per the neighbouring methods now.